IFC_Adapter/IFC_Adapter.csproj

```diff
@@ -0,0 +1,258 @@
+    </Reference>
+    <Reference Include="Xbim.IO.MemoryModel, Version=5.1.0.0, Culture=neutral, PublicKeyToken=11e3655e576ec5a9, processorArchitecture=MSIL">
+      <HintPath>..\packages\Xbim.IO.MemoryModel.5.1.323\lib\net46\Xbim.IO.MemoryModel.dll</HintPath>
+    </Reference>
+    <Reference Include="Xbim.ModelGeometry.Scene, Version=5.1.0.1, Culture=neutral, PublicKeyToken=11e3655e576ec5a9, processorArchitecture=MSIL">
+      <HintPath>..\packages\Xbim.ModelGeometry.Scene.5.1.403\lib\net472\Xbim.ModelGeometry.Scene.dll</HintPath>
+    </Reference>
+    <Reference Include="Xbim.Tessellator, Version=5.1.0.0, Culture=neutral, PublicKeyToken=11e3655e576ec5a9, processorArchitecture=MSIL">
+      <HintPath>..\packages\Xbim.Tessellator.5.1.323\lib\net47\Xbim.Tessellator.dll</HintPath>
+    </Reference>
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="AdapterActions\Execute.cs" />
+    <Compile Include="AdapterActions\Remove.cs" />
+    <Compile Include="AdapterActions\Push.cs" />
+    <Compile Include="AdapterActions\Pull.cs" />
+    <Compile Include="CRUD\Read.cs" />
+    <Compile Include="IFCAdapter.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\IFC_Engine\IFC_Engine.csproj">
+      <Project>{41f37d36-a8e4-46b9-b59e-e1504280ce44}</Project>
+      <Name>IFC_Engine</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\IFC_oM\IFC_oM.csproj">
+      <Project>{506679c5-d5b2-4758-b1dd-b980f198cbd6}</Project>
+      <Name>IFC_oM</Name>
+    </ProjectReference>
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="packages.config" />
+  </ItemGroup>
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <PropertyGroup>
+    <PostBuildEvent>
+		xcopy "$(TargetDir)$(TargetFileName)"  "C:\ProgramData\BHoM\Assemblies" /Y
+		xcopy "$(TargetDir)Microsoft.Extensions.Logging.Abstractions.dll"  "C:\ProgramData\BHoM\Assemblies" /Y
+		xcopy "$(TargetDir)Microsoft.Extensions.Logging.dll"  "C:\ProgramData\BHoM\Assemblies" /Y
+		xcopy "$(TargetDir)Microsoft.Extensions.Options.dll"  "C:\ProgramData\BHoM\Assemblies" /Y
+		xcopy "$(TargetDir)Xbim.Common.dll"  "C:\ProgramData\BHoM\Assemblies" /Y
+		xcopy "$(TargetDir)Xbim.Geometry.Engine.Interop.dll"  "C:\ProgramData\BHoM\Assemblies" /Y
+		xcopy "$(TargetDir)Xbim.Geometry.Engine32.dll"  "C:\ProgramData\BHoM\Assemblies" /Y
+		xcopy "$(TargetDir)Xbim.Geometry.Engine64.dll"  "C:\ProgramData\BHoM\Assemblies" /Y
+		xcopy "$(TargetDir)Xbim.Ifc.dll"  "C:\ProgramData\BHoM\Assemblies" /Y
+		xcopy "$(TargetDir)Xbim.Ifc2x3.dll"  "C:\ProgramData\BHoM\Assemblies" /Y
+		xcopy "$(TargetDir)Xbim.Ifc4.dll"  "C:\ProgramData\BHoM\Assemblies" /Y
+		xcopy "$(TargetDir)Xbim.IO.MemoryModel.dll"  "C:\ProgramData\BHoM\Assemblies" /Y
+		xcopy "$(TargetDir)Xbim.ModelGeometry.Scene.dll"  "C:\ProgramData\BHoM\Assemblies" /Y
+		xcopy "$(TargetDir)Xbim.Tessellator.dll"  "C:\ProgramData\BHoM\Assemblies" /Y
```

Comment:
I'm sure you already checked, but to confirm, these are 100% necessary right?

Reply:
Ja, unfortunately : < As discussed, happy to keep it this way for now and revisit when becomes a problem (i.e. starts being considered to be added to the installer 🙈)